### DISCUSSION TITLE
关于迅雷云盘文件下载的问题

### DISCUSSION THREAD

**Discussion by crazynao** [AlistGo/alist#1120] [Q&A] [answered]
为什么迅雷网盘的文件点击下载之后下载的文件名会变成“下载”
### 👇👇👇
[https://pic.28a.xyz/2022/05/18/1b79f2f663aac.png](url)

**Comment by anwen-anyi** [comment, marked as answer]
> 为什么迅雷网盘的文件点击下载之后下载的文件名会变成“下载”
> 
> ### 👇👇👇
> 
> [https://pic.28a.xyz/2022/05/18/1b79f2f663aac.png](url)

迅雷就是这样…你可以右上角复制直链然后去IDM手动添加任务就好了 或者去其他下载的地方添加刚刚复制的直链就好
